src/test/java/org/apache/commons/lang3/exception/ExceptionUtilsTest.java

```diff
@@ -850,4 +850,40 @@ public void testWrapAndUnwrapThrowable() {
         final Throwable t = assertThrows(Throwable.class, () -> ExceptionUtils.wrapAndThrow(new TestThrowable()));
         assertTrue(ExceptionUtils.hasCause(t, TestThrowable.class));
     }
+
+    @Test
+    public void testIsChecked_unchecked() {
+        final boolean result = ExceptionUtils.isChecked(new IllegalArgumentException());
```

Comment:
@orionlibs 
There is no need for a single-use local variable here and in other new tests.

Reply:
@garydgregory so, should I create new local variables for each case?

Reply:
@garydgregory what should I do for this PR to be successful?

Reply:
> @garydgregory so, should I create new local variables for each case?

You've got it backward.

Reply:
@garydgregory  so, I don't need to create local variables if the assertion looks like this:
assertTrue(ExceptionUtils.isChecked(new IllegalArgumentException()));
Is that what you mean?

Reply:
👍😉

Reply:
done!

Reply:
Not quite, see https://github.com/apache/commons-lang/pull/1069#issuecomment-1616686083

Reply:
@garydgregory you are right. I made ConcurrentUtils to use the new method. I hope this completes this PR. Thank you for pointing that comment out. I focused on other parts and I missed that part of the comment. Distractions are bad